src/components/bars/ProgressBar.js

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useState } from 'react';
+import '../../styles/components/bars/ProgressBar.css';
+
+const ProgressBar = ({ startTime }) => {
+    const today = new Date();
+
+    const checkHour = () => {
+        const shour = parseInt(startTime.sHour);
+
+        if (startTime.status === 'FAST') {
+            if (shour + 1 >= 24) return shour - 23;
+            return shour + 1;
+        }
+        else {
+            if (shour + 14 >= 24) return shour - 10;
+            return shour + 14;
+        }
+    }
+    const checkLeftTime = () => {
+        const currentHour = today.getHours();
+        const currentMinute = today.getMinutes();
+        let endHour = endTime.eHour;
```

Comment:
It would be better to put it above the part that defines the part that declares the variable.

Reply:
thanks. I'll fix it.